packages/app/src/game/module/board/overlays/readyOverlay.tsx

```diff
@@ -62,10 +84,21 @@ const ReadyOverlay: React.FunctionComponent = () => {
 						</div>
 						<div className="player-picture">
 							<PlayerAvatar player={opponent} />
+							<QuickChatBox sendingPlayerId={receivingPlayerId} />
 						</div>
 					</div>
 				</div>
 				<HeadToHeadStats player={localPlayer} opponent={opponent} />
+				<div className="quick-chat-box-container">
```

Comment:
Looking at the markup here I wonder what's the point of the inner `div` (`.quick-chat-buttons`) at all

Why not just get rid of the container, put the `quick-chat-buttons` up a level, and then just use flex inside there to space the buttons out (probably `justify-content`)